### ISSUE TITLE
Spurious text in politeElementProperty

### ISSUE THREAD

**Issue by samreid** [phetsims/balloons-and-static-electricity#285]
Starting up Balloons and Static Electricity with the json phet-io log, I see the following messages before I interact with the sim:

![image](https://user-images.githubusercontent.com/679486/29031833-bafb35a4-7b4c-11e7-901a-8c9dc65277f2.png)

In particular, I'm worried about why it keeps switching back and forth between "" and "Released".  @jessegreenberg can you please investigate?

**Comment by jessegreenberg** [comment]
Thanks, we shouldn't be seeing this on startup.

> In particular, I'm worried about why it keeps switching back and forth between "" and "Released".

By default, the previous alert content is cleared from the HTML element with the ```aria-live``` attribute so that if the same alert is announced twice the screen reader will still read it. For instance, in a stream of alerts like

> "Released".
> "Moving slowly towards sweater"
> "Still moving towards sweater"
> "Still moving towards sweater"
> "Still moving towards sweater"

a screen reader will likely ignore the second two alerts unless we empty the content first.

**Comment by jessegreenberg** [comment]
The above commit should prevent alerts from coming through during startup. But when politeElementProperty changes, we will still generally see it toggle to "", then to "Whatever alert comes through the AT".  @samreid is this OK?

**Comment by samreid** [comment]
@jessegreenberg it seems like the transient `""` is an implementation detail that shouldn't be part of the public API or data stream.  Have you considered using an Emitter as an alternative to Property?  If so, would it be possible only to emit the text changes (or when it is intentionally set to `""`, but to avoid emitting when it is set to `""` as a transient?  

Alternatively, it may be nice to have the Property track the value that the aria should be reading and then implement the transient `""` separately.  Can you please investigate this first?

**Comment by jessegreenberg** [comment]
@samreid sure that makes sense, I'll take a look at clearing without updating the Property.

**Comment by jessegreenberg** [comment]
> Alternatively, it may be nice to have the Property track the value that the aria should be reading and then implement the transient "" separately.

This is pretty straight forward.  Is it OK that the Property ```oldValue``` is usually ```""```?

**Comment by jessegreenberg** [comment]
Handled in the above commit, when an alert comes through it looks like this in the formatted data stream:
<img width="1190" alt="screen shot 2017-08-07 at 1 12 15 pm" src="https://user-images.githubusercontent.com/6396244/29042044-93ccc5c6-7b72-11e7-8203-f60c5b6e2023.png">

We shouldn't see things like
```ariaHerald.politeElementProperty changed { "oldValue": "My alert", "newValue": "" }```
anymore.

I think this can be closed, but is there anything else @samreid?

**Comment by samreid** [comment]
Is the politeElement ever being "cleared" by setting the value to `""`?  I'm not asking about a transient value before another message comes, but setting the state to be blank more semi-permanently (if nothing is happening)?

**Comment by samreid** [comment]
Also, in the commit above I cannot see where the `politeElementProperty` is no longer receiving the `""` value, is there another commit that isn't linked to this issue?

**Comment by jessegreenberg** [comment]
> Is the politeElement ever being "cleared" by setting the value to ""

Yes, its possible.  AriaHerald has functions like ```clearAssertive``` and ```clearAll``` that explicitly set the content to ```""``` when necessary.  ```AriaHerald.announcePolite( "" )``` will have the same effect.

> I cannot see where the politeElementProperty is no longer receiving the ""

In https://github.com/phetsims/scenery-phet/commit/37bf8502c5f70a2e3a1fd7adae50020b7546b321, we are clearing the old HTML content directly without going through the Property so that it doesn't show up in the data stream.

      // clearing the old content allows repeated alerts, but clear the HTMLElement content directly
      // so that we don't see the empty content in the PhET-iO data stream
      if ( withClear ) { liveElement.textContent = ''; }

Then we update the content with the new alert by going through the Property.

**Comment by samreid** [comment]
> I cannot see where the politeElementProperty is no longer receiving the ""

I see, it is no longer resetting.  All seems well! Closing.
